src/ser/mod.rs

```diff
@@ -156,6 +156,12 @@ impl Serializer {
                 .extend((0..pretty.indent).map(|_| config.indentor.as_str()));
         }
     }
+
+    fn serialize_escaped_str(&mut self, value: &str) {
+        self.output += "\"";
+        self.output += &value.chars().flat_map(|c| c.escape_default()).collect::<String>();
```

Comment:
I would avoid the string allocation and immediately push the characters to the output.

Reply:
Err right, I was trying it out and forgot about it then. Thanks!